app/src/hooks/useBioHubApi.ts

```diff
@@ -1,37 +1,10 @@
 import { useKeycloak } from '@react-keycloak/web';
```

Comment:
Everything removed from this file was only recently added, to solve the same problem this PR is solving, but in a worse way.  All of the main logic here has more-or-less moved to the configContext.tsx.